src/Utils/ASTDefinitionBuilder.php

```diff
@@ -407,13 +407,13 @@ private function makeUnionDef(UnionTypeDefinitionNode $def): UnionType
             // Note: While this could make assertions to get the correctly typed
             // values below, that would throw immediately while type system
             // validation with validateSchema() will produce more actionable results.
-            'types'       => function () use ($def): array {
-                return Utils::map(
-                    $def->types,
-                    function ($typeNode): Type {
-                        return $this->buildType($typeNode);
-                    }
-                );
+            'types'       =>  function () use ($def): array {
```

Comment:
ok, CS should have caught this additional space :/

Reply:
Agree

Reply:
```suggestion
            'types'       => function () use ($def): array {
```